.circleci/integration-tests/script.sh

```diff
@@ -22,14 +28,37 @@ function clean() {
   find  . -name "example_*" -exec rm {} \;
 }
 
-if [ "$1" == "-h" ] || [ "$#" -ne 3 ]; then
+if [ "$1" == "-h" ]; then
   echo_help
   exit
 fi
 
-BASE_DOMAIN=$1
-RELEASE_NAME=$2
-ASTRO_API_KEY=$3
+DEPLOYMENT_INSTANCE=$1
+# Variable for Staging deployment
+DOCKER_REGISTRY=""
+RELEASE_NAME=""
+ASTRONOMER_API_KEY=""
+# Variable for Gen2 deployment
+ORGANIZATION_ID=""
+DEPLOYMENT_ID=""
+ASTRONOMER_KEY_ID=""
+ASTRONOMER_KEY_SECRET=""
+
+if [[ ${DEPLOYMENT_INSTANCE} == "STAGING" ]]; then
+  DOCKER_REGISTRY=$2
+  RELEASE_NAME=$3
+  ASTRONOMER_API_KEY=$4
```

Comment:
Can we add comments at the top of the script to describe what each of the positional arguments are meant for? 
Something like the following:
```
# The script accepts the following arguments: 
# $1 - Deployment instance
# $2 - Docker registry URL
# $3 - Astronomer API Key
.
.
.
```

So that we get a quick view of the arguments and their order that can be passed to the script.